src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -1,14 +1,35 @@
 import React, { useEffect, useState, useContext } from 'react';
+import styled from 'styled-components';
 import 'isomorphic-fetch';
 import { string } from 'prop-types';
 import { MostRead } from '@bbc/psammead-most-read';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MAX,
+  GEL_GROUP_5_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_MARGIN_ABOVE_400PX,
+  GEL_MARGIN_BELOW_400PX,
+} from '@bbc/gel-foundations/spacings';
 import { ServiceContext } from '#contexts/ServiceContext';
 import webLogger from '#lib/logger.web';
 import { mostReadRecordIsFresh, shouldRenderLastUpdated } from '../utilities';
 import LastUpdated from './LastUpdated';
 
 const logger = webLogger();
 
+const StyledMostRead = styled(MostRead)`
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
```

Comment:
Do you need the max-width in here?

Reply:
The below media query `@media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {` applies different styles after this max so it just avoids them overriding. We're putting this one on hold till tomorrow so we can Jim some questions. We might not need Group 5 in the new layout.